emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java

```diff
@@ -99,13 +128,16 @@
     popupWindow = new PopupWindow(context);
 
     final OnEmojiLongClickListener longClickListener = new OnEmojiLongClickListener() {
-      @Override public void onEmojiLongClick(@NonNull final EmojiImageView view, @NonNull final Emoji emoji) {
+      @Override
+      public void onEmojiLongClick(@NonNull final EmojiImageView view, @NonNull final Emoji emoji) {
         variantPopup.show(view, emoji);
       }
     };
 
     final OnEmojiClickListener clickListener = new OnEmojiClickListener() {
-      @Override public void onEmojiClick(@NonNull final EmojiImageView imageView, @NonNull final Emoji emoji) {
+      @Override
+      public void onEmojiClick(@NonNull final EmojiImageView imageView,
+          @NonNull final Emoji emoji) {
```

Comment:
Here for example.

Reply:
Ok, I imagine this is because of chars limit in a line. Will try to get to it soon!

Reply:
Great, thanks!

Reply:
Can you let me know the char limit per line that you folks use?

Reply:
Personally I use 120, but this project doesn't use one I think, the lines can be as long as they want (At least I have seen lines longer then my configured limit in the IDE). I think it would be good to introduce a char limit (and update the style in general), but that should be a separate PR.